src/main/java/org/moera/node/global/VirtualPageInterceptor.java

```diff
@@ -35,16 +35,12 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
         VirtualPageHeader.put(response, virtualPage.value());
 
         if (requestContext.isBrowserExtension()) {
-            response.setContentType("text/html; charset=utf-8");
+            response.setContentType("text/plain; charset=utf-8");
             Writer out = new OutputStreamWriter(response.getOutputStream());
-            out.write("<!DOCTYPE html>\n"
-                    + "<html>\n"
-                    + "    <head>\n"
-                    + "        <meta name='viewport' content='width=device-width, initial-scale=1, shrink-to-fit=no'>\n"
-                    + "        <title>Moera</title>\n"
-                    + "    </head>\n"
-                    + "    <body></body>\n"
-                    + "</html>\n");
+            out.write("Moera client is loading.\n"
+                    + "If you are seeing this message for some time now, something is not right.\n"
+                    + "If Moera extension settings are opened, try closing them.\n"
+                    + "It's not helping? Contact developers though github: https://github.com/MoeraOrg/moera-issues\n");
```

Comment:
Let's not scary user with long explanations what may go wrong ;) Most of the time the loading will work as expected and the extension storage locking bug we will fix in the near future. So let's display just "Loading Moera client..."

Reply:
If user has time to read it, i think something did go wrong 😃

Reply:
I think we should just change it to `If you are seeing this message for a couple of minutes` and change formatting a bit